packages/veritone-react-common/src/components/ProfileMenu/index.js

```diff
@@ -31,6 +31,9 @@ class ProfileMenu extends React.Component {
         firstName: string,
         lastName: string,
         image: string
+      }),
+      organization: shape({
+        organizationId: number
```

Comment:
Remove unused prop

Reply:
i can remove here - but user is passed through